### PR TITLE
[Hub Generated] Review request for Microsoft.Billing to add version preview/2018-11-01-preview

### PR DESCRIPTION
**Pull request by sarang-amondikar**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/sarang-amondikar/azure-rest-api-specs/dev-billing-Microsoft.Billing-2018-11-01-preview...master/:tab?).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/6403 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/6403/commits/f9cbc296845932be678ed9a95adce1b39d765b8f

File: specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json [hunk 1]
```diff
@@ -150,6 +150,50 @@
...
+        ],
+        "x-ms-examples": {
+          "ValidateAddressValid": {
+            "$ref": "./examples/ValidateAddressValid.json"
+          },
+          "ValidateAddressInvalid": {
+            "$ref": "./examples/ValidateAddressInvalid.json"
+          }
+        },
+        "operationId": "ValidateAddress_Validate",
```
Comment (jhendrixMSFT) [inline specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json]: Better name would be `Addresses_Validate`.

**Review by wilcobmsft — APPROVED** [review]

File: specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json [hunk 2]
```diff
@@ -150,28 +150,25 @@
         }
       }
     },
-    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/validateAddress": {
+    "/providers/Microsoft.Billing/validateAddress": {
```
Comment (RyanBensonMSFT) [inline specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json]: Based on the swagger repo, it looks like .../validateAddress is only present in this -preview api. Can you confirm we're not renaming a stable api here.

**Review by ryansbenson — COMMENTED** [review]
Approved from ARMs side

**Review by jhendrixMSFT — APPROVED** [review]

**Comment by jhendrixMSFT** [comment]
@sarang-amondikar has this API been deployed?

**Comment by sarang-amondikar** [comment]
@jhendrixMSFT the api  is not yet deployed, we want to test it in ppe first

**Comment by jhendrixMSFT** [comment]
Sounds good, please update when the API is live so we can merge.

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Comment by sarang-amondikar** [comment]
the api is now deployed to prod, could you please approve this PR?

**Pull request merged**